src/main/fulcro/client.cljc

```diff
@@ -184,6 +184,21 @@
                       [k valid]))
      :clj  {}))
 
+(defn- mutation-query? [tx]
+  (boolean (some #(util/mutation? %) tx)))
+
+(defn- split-parser
+  "Generate a parser that splits reads and writes across two parsers: the supplied query parser for queries, and the built-in
+  parser for mutations."
+  [query-parser]
+  (let [mutation-parser (prim/parser {:read (constantly nil) :mutate app/write-entry-point})]
+    (fn split-parser*
+      ([env query target]
+       (if (mutation-query? query)
+         (mutation-parser env query target)
+         (query-parser (assoc env :target target) query)))
```

Comment:
I thought we would only send the custom parser when there is no target, we changed the assumption or was a miss?

Reply:
this is just to make sure there is a target on there...it will be nil unless I've screwed up

Reply:
I guess it could just be dropped...I was thinking ppl would expect to see it, but probably not, actually